.github/workflows/contribs.yml

```diff
@@ -11,7 +16,7 @@ jobs:
       - name: Contribs
         uses: carlescufi/action-contribs@d139e12dabbe631fcc94ec3e19e386a11d3e1a26 # main
         with:
-          github-token: ${{ secrets.NCS_GITHUB_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Did you test if secrets.GITHUB_TOKEN permissions can be elevated with permissions+prs+write? If I remember correctly github restricts GITHUB_TOKEN permission automatically from forks - so my assumption is this does not work. Please test carefully.